gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixTask.java

```diff
@@ -67,113 +57,63 @@
  *   {@link org.apache.gobblin.runtime.Task}(s), it persists the {@link TaskState} of each {@link org.apache.gobblin.runtime.Task} to
  *   a file that will be collected by the {@link GobblinHelixJobLauncher} later upon completion of the job.
  * </p>
- *
- * @author Yinan Li
  */
 @Alpha
 public class GobblinHelixTask implements Task {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(GobblinHelixTask.class);
+  private static final Logger _logger = LoggerFactory.getLogger(GobblinHelixTask.class);
```

Comment:
Should the underscore be removed for code consistency?

Reply:
See comment above.